### PR TITLE
Force patch version for gRPC builds

### PR DESCRIPTION
**Pull request by ignasi35**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -86,49 +86,49 @@ jobs:
...
       script: "cd mixed-persistence/mixed-persistence-java-sbt && sbt test stage"
     - name: "Run tests Mixed Persistence (scala)"
-      env: TRAVIS_JDK=8
+      env: TRAVIS_JDK=~1.8.0-0
       script: "cd mixed-persistence/mixed-persistence-scala-sbt && sbt test stage"
 
     # gRPC Samples
     - name: "Run tests gRPC example (java)"
-      env: TRAVIS_JDK=8
+      env: TRAVIS_JDK=adopt@1.8.0-242
```
Comment (ihostage) [inline .travis.yml]:
```suggestion
      env: TRAVIS_JDK=1.8.0-242
```

File: .travis.yml [hunk 2]
```diff
@@ -86,49 +86,49 @@ jobs:
...
       script: "cd grpc-example/grpc-example-java && sbt test stage"
     - name: "Run tests gRPC example (scala)"
-      env: TRAVIS_JDK=8
+      env: TRAVIS_JDK=adopt@1.8.0-242
```
Comment (ihostage) [inline .travis.yml]:
```suggestion
      env: TRAVIS_JDK=1.8.0-242
```

**Review by ihostage — COMMENTED** [review]
`TRAVIS_JDK` used in command
```
jabba ls-remote "adopt@${TRAVIS_JDK}" --latest=patch
```
that already has a prefix `adopt@`

**Comment by ignasi35** [comment]
Hopefully https://github.com/playframework/play-grpc/pull/264 will make this PR obsolete.

**Review by ihostage — APPROVED** [review]

**Review by raboof — APPROVED** [review]

**Pull request merged**